.github/workflows/go.yml

```diff
@@ -75,60 +75,68 @@ jobs:
     name: e2e tests(oap+ui+agent)
     runs-on: ubuntu-latest
     steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v3
+      - name: free disk space
+        run: bash .github/free-disk-space.sh 
       - name: Install Go
         uses: actions/setup-go@v3
         with:
           go-version: '1.20'
         id: go
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v3
       - name: Run E2E Test(oap+ui+agent)
-        uses: apache/skywalking-infra-e2e@996ed8902e941e2883fcf0ac5b3090364942d205      # always prefer to use a revision instead of `main`.
+        uses: apache/skywalking-infra-e2e@main      # always prefer to use a revision instead of `main`.
```

Comment:
We should lock the version. You could bump up the commit ID, rather than the `main` branch.

Reply:
Make sense to me.